moveit_kinematics/ikfast_kinematics_plugin/templates/ikfast61_moveit_plugin_template.cpp

```diff
@@ -578,29 +578,29 @@ void IKFastKinematicsPlugin::getSolution(const IkSolutionList<IkReal>& solutions
 
 double IKFastKinematicsPlugin::harmonize(const std::vector<double>& ik_seed_state, std::vector<double>& solution) const
 {
-  double dist_sqr = 0;
+  double dist_abs = 0;
   std::vector<double> ss = ik_seed_state;
   for (size_t i = 0; i < ik_seed_state.size(); ++i)
   {
-    while (ss[i] > 2 * M_PI)
+    while (ss[i] > M_PI)
     {
       ss[i] -= 2 * M_PI;
     }
-    while (ss[i] < 2 * M_PI)
+    while (ss[i] < -1 * M_PI)
     {
       ss[i] += 2 * M_PI;
     }
-    while (solution[i] > 2 * M_PI)
+    while (solution[i] > M_PI)
     {
       solution[i] -= 2 * M_PI;
     }
-    while (solution[i] < 2 * M_PI)
+    while (solution[i] < -1 * M_PI)
     {
       solution[i] += 2 * M_PI;
     }
-    dist_sqr += fabs(ik_seed_state[i] - solution[i]);
+    dist_abs += fabs(ss[i] - solution[i]);
   }
-  return dist_sqr;
+  return dist_abs;
```

Comment:
could you elaborate in your PR description what these changes are for? was this ``harmonize()`` function not being used previously? was it this broken?